tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -152,4 +186,11 @@ jobs:
         buildConfig: $(buildConfig)
         artifactName: 'onnxruntime-win-gpu-$(buildArch)'
 
+    - task: PowerShell@2
+      displayName: 'Clean up Cuda Path 9.1'
+      inputs:
+        targetType: 'filePath'
+        filePath: '$(Build.SourcesDirectory)/tools/ci_build/github/windows/clean_up_cuda_prop_files.ps1'
```

Comment:
Where would you run this script?
Can we avoid to do that? 

I manually installed the extensions to our github CI build machines, but it often get deleted by scripts like this.

Reply:
let's revisit this after tomorrow's release.